src/target/riscv/riscv-013.c

```diff
@@ -1726,6 +1777,12 @@ static int examine(struct target *target)
 		dmi_write(target, DM_DMCONTROL, DM_DMCONTROL_DMACTIVE);
 		dm->was_reset = true;
 	}
+
+	if (dbgbase_verify(target) == false) {
+		LOG_ERROR("dbgbase 0x%x verify fail\n", target->dbgbase);
```

Comment:
This error message can be improved, so that the users can understand it better. For example:

```suggestion
		LOG_TARGET_ERROR(target, "Could not find debug module with DMI base address (dbgbase) = 0x%x", target->dbgbase);
```

We should use `LOG_TARGET_*` instead of `LOG_*` where possible.

Also don't include `\n` at the end of the messages. The `LOG_*` functions print the newline automatically.